app/views/projects/issues/index.html.erb

```diff
@@ -5,11 +5,28 @@
   project: current_project
 } %>
 
+<% if @open_issue.present? %>
+  <%= render partial: 'issues/issue', locals: {
```

Comment:
Usually this `_entity` is used for listing purposes. Maybe it's better to call it `_issue_detail`, `_editable_issue_overlay`, `_editable_issue_dialog`..

Reply:
Let's go with `issue_detail`!